test/ccd2rgb.jl

```diff
@@ -0,0 +1,31 @@
+@testset "ccd2rgb" begin
+    if !isfile("data/casa_0.5-1.5keV.fits")
+        mkpath("data")
+        download("http://chandra.harvard.edu/photo/2009/casa/fits/casa_0.5-1.5keV.fits", "data/casa_0.5-1.5keV.fits")
+    end
+    if !isfile("data/casa_1.5-3.0keV.fits")
+        mkpath("data")
+        download("http://chandra.harvard.edu/photo/2009/casa/fits/casa_1.5-3.0keV.fits", "data/casa_1.5-3.0keV.fits")
+    end
+    if !isfile("data/casa_4.0-6.0keV.fits")
+        mkpath("data")
+        download("http://chandra.harvard.edu/photo/2009/casa/fits/casa_4.0-6.0keV.fits", "data/casa_4.0-6.0keV.fits")
+    end
+    if !isfile("data/ccd2rgd.jld")
+        mkpath("data")
+        download("https://bintray.com/aquatiko/AstroImages.jl/download_file?file_path=ccd2_rgb.jld","data/ccd2rgb.jld")
+    end
+        
+    r = FITS("data/casa_0.5-1.5keV.fits")[1]
+    b = FITS("data/casa_1.5-3.0keV.fits")[1]
+    g = FITS("data/casa_4.0-6.0keV.fits")[1]
+    linear_res = ccd2rgb(r, b, g, shape_out = (1000,1000))
+    log_res = ccd2rgb(r, b, g, shape_out = (1000,1000), stretch = log)
+    sqrt_res = ccd2rgb(r, b, g, shape_out = (1000,1000), stretch = sqrt)
+    asinh_res = ccd2rgb(r, b, g, shape_out = (1000,1000), stretch = asinh)
+
+    @test all(linear_res .=== load("data/ccd2rgb.jld", "linear"))
```

Comment:
Try doing approximate comparison, in general it's now a wise idea to test identity (or even worse "equality") when doing calculations with floating point numbers.  This *might* solve the issues in the tests